applications/MappingApplication/custom_utilities/interface_object_manager_base.h

```diff
@@ -131,248 +124,244 @@ namespace Kratos
           int all_neighbors_found = 1; // set to "1" aka "true" by default in case
                                        // this partition doesn't have a part of the interface!
 
-          for (auto& interface_obj : m_interface_objects) {
-              if (!interface_obj->NeighborFound()) {
+          for (auto& interface_obj : mInterfaceObjects) {
+              if (!interface_obj->NeighborOrApproximationFound()) { // TODO this prevents from printing warnings in case only an approx is found
                   all_neighbors_found = 0;
               }
           }
 
           // This is necessary bcs not all partitions would start a new search iteration!
-          m_model_part.GetCommunicator().MinAll(all_neighbors_found);
+          mrModelPart.GetCommunicator().MinAll(all_neighbors_found);
 
           return all_neighbors_found;
       }
 
-      virtual void CheckResults() = 0;
+      void CheckResults() {
+          for (auto& interface_obj : mInterfaceObjects) {
+              const int pairing_status = interface_obj->GetPairingStatus();
+              if (pairing_status == 0) { // TODO
+                  std::cout << "MAPPER WARNING, Rank " << mCommRank
+                            << "\tPoint [ "
+                            << interface_obj->X() << " | "
+                            << interface_obj->Y() << " | "
+                            << interface_obj->Z() << " ] " 
+                            << "has not found a neighbor" << std::endl;
+              } else if (pairing_status == 1) { // TODO
+                  std::cout << "MAPPER WARNING, Rank " << mCommRank
+                            << "\tPoint [ "
+                            << interface_obj->X() << " | "
+                            << interface_obj->Y() << " | "
+                            << interface_obj->Z() << " ] " 
+                            << "uses an approximation" << std::endl;
+              }
+          }
+      }
 
       InterfaceObjectConfigure::ContainerType& GetInterfaceObjects() {
-          return m_interface_objects;
+          return mInterfaceObjects;
       }
 
       // ***** InterfaceObjectManagerSerial *****
-      virtual void GetInterfaceObjectsSerialSearch(InterfaceObjectConfigure::ContainerType& candidate_send_objects) {
-          KRATOS_ERROR << "MappingApplication; InterfaceObjectManagerBase; "
-                       << "\"GetInterfaceObjectsSerialSearch\" of the base class called!"
-                       << std::endl;
+      virtual void GetInterfaceObjectsSerialSearch(InterfaceObjectConfigure::ContainerType& rCandidateSendObjects) {
+          KRATOS_ERROR << "Base class function called!" << std::endl;
       }
```

Comment:
Same reason as above, I dont need every of those functions for every child class